codeexamples/SYCLUsingCUDAStaticLibraryOfKernels/StaticLibraryBuildCUDAKernels/src/library.cu

```diff
@@ -0,0 +1,20 @@
+// In-house headers:
+#include "staticlibfnscudakernelsDXTC.h"
+#include "cudakernels.cu"
+
+namespace StaticLibFnsCUDAKernelsDXTC
+{
+    cudaError_t CompressFacade( const int vBlocksPerLaunch, const uint vBlocks, 
+                                const uint *vpPermutations, const uint *vpImage, uint *vpResult, const int vBlockOffset ) 
+    {
+        compress<<< min( vBlocksPerLaunch, vBlocks - vBlockOffset ), NUM_THREADS >>>(
+              vpPermutations, vpImage, (uint2 *) vpResult, vBlockOffset );
+
+        // Interop with host_task doesn't add CUDA event to task graph
+        // so we must manually sync here.
+        cudaDeviceSynchronize();
+
+        return cudaGetLastError();
+    }
+
+} // namespace StaticLibFnsCUDAKernelsDXTC
```

Comment:
This file is missing a newline at the end

Reply:
I am not aware this is problem. What issue could this course?

Reply:
It used to be disallowed for C++ source files. Now it is but it can be strange with tools that expect "normal" text files (which end with a newline followed by an EOF). Typically Microsoft tools haven't enforced it. If we had clang-format (which I guess we'll add later) then it would force the newlines at the end.

Reply:
Change made. Done.